.github/workflows/release.yml

```
@@ -0,0 +1,22 @@
+name: NPM publish on release 
+on: 
+  push: 
+    tags: 
+      - v[0-9].[0-9].[0-9]
+
+jobs: 
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+    with:
+      node-version: 16
+      registry-url: 'https://registry.npmjs.org'
+    - run: npm install
+    - run: |
+        npm prepack
+        npm publish 
+      env: 
+        GITHUB_WORKSPACE: ${GITHUB_WORKSPACE}
+        NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

```suggestion
    - name: Publish package to npm.js
      uses: JS-DevTools/npm-publish@v1
      with:
        token: ${{ secrets.NPM_TOKEN }}
        access: public
```

DevOps is using this component to build & publish js packages to npm, please consider replacing publish step to this one.

---

sure Donny will replace